controllers/errors-controller.js

```diff
@@ -2,4 +2,21 @@ const invalidPathError = (req, res, next) => {
   res.status(404).send({ msg: 'Sorry, path not found' });
 };
 
-module.exports = { invalidPathError };
+const handleCustomErrors = (err, req, res, next) => {
+  if (err === 'Sorry, no article found') {
+    res.status(404).send({ msg: err });
+  }
+  if (err === '22P02') {
```

Comment:
Line 9: This line exists already in your PSQL handler, you made need to check if (err.status && err.msg) exist to enter your custom handler, and check if (err.code) exists to enter your PSQL handler